buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -662,7 +662,7 @@ class BeamModulePlugin implements Plugin<Project> {
         proto_google_cloud_datastore_v1             : "com.google.api.grpc:proto-google-cloud-datastore-v1", // google_cloud_platform_libraries_bom sets version
         proto_google_cloud_firestore_v1             : "com.google.api.grpc:proto-google-cloud-firestore-v1", // google_cloud_platform_libraries_bom sets version
         proto_google_cloud_pubsub_v1                : "com.google.api.grpc:proto-google-cloud-pubsub-v1", // google_cloud_platform_libraries_bom sets version
-        proto_google_cloud_pubsublite_v1            : "com.google.api.grpc:proto-google-cloud-pubsublite-v1", // google_cloud_platform_libraries_bom sets version
+        proto_google_cloud_pubsublite_v1            : "com.google.api.grpc:proto-google-cloud-pubsublite-v1:1.5.0", // TODO(dpcollins-google): Let google_cloud_platform_libraries_bom set version once high enough
```

Comment:
Sorry a couple more suggestions here:
- Could we also put a TODO by `google_cloud_platform_libraries_bom` so whoever upgrades it knows to check on this?
- We should run the linkage checker for this dependency upgrade (described in detail here: https://cwiki.apache.org/confluence/display/BEAM/Java+Tips#JavaTips-DependencyUpgrades)

Reply:
TODO added